### DISCUSSION TITLE
https://travellings.link/无法打开

### DISCUSSION THREAD

**Discussion by coolf233** [travellings-link/travellings#1139] [寻求帮助] [answered]
https://[travellings.link](https://travellings.link/)/连接已经无法打开请及时修复

**Comment by Big-Cake-jpg** [comment, marked as answer]
> https://[travellings.link](https://travellings.link/)/连接已经无法打开请及时修复

请使用 travellings.cn 或更换 DNS

**Comment by imjason** [comment]
经检查，部分地区异常，可以使用travellings.cn缓解

**Comment by Crazyokd** [comment]
我两个链接（https://travellings.link/ 和 https://travellings.cn）都不行了

**Comment by Big-Cake-jpg** [comment]
对于 DNS 污染，请设置一个不容易被污染的 DNS 服务器，如 223.5.5.5 / 8.8.8.8 / 4.2.2.2

**Comment by imjason** [comment]
dns使用的是华为云，HK镜像服务器出现异常正在处理
经核实，服务器正在迁移到新机柜预计8/16修复    8/15 9:29

**Comment by imjason** [comment]
经修复开往HK镜像恢复正常 8/17 6:04
